coordinator/quote/ert.go

```diff
@@ -26,6 +26,8 @@ type PackageProperties struct {
 	ProductID *uint64
 	// Security version number of the package.
 	SecurityVersion *uint
+	// Accepted TCB levels
+	AcceptedTCBLevels []string
```

Comment:
Maybe we should call this AcceptedTCBStatuses because it's not only about the level, but also about configuration and sw hardening.

Reply:
You are right, I will rename it, thanks for the note. Should I do it in a new separate commit or squash it?

Reply:
If it's ok for you to edit both of your commits (it should be renamed in the CLI, too), please do so. If that's too much effort, you can also do a separate commit.